### PR TITLE
Add test for CLI tool

### PR DESCRIPTION
**Pull request by mhmohona**
### Contributor checklist

- [x] This pull request is on a [separate branch](https://docs.github.com/en/get-started/quickstart/github-flow) and not the main branch

---

### Description

image

### Related issue

- #ISSUE_NUMBER

### CODE REVIEW

**Comment by wkyoshida** [comment]
Hey, @mhmohona! :wave: 

I noticed the tests appear to be failing when running in the CI. Did notice also though that the screenshot in the PR description shows the tests do run OK when `pytest` is ran within `src/scribe_data/cli`. Does running `pytest` in the root of the project still run OK too?

**Comment by mhmohona** [comment]
> Hey, @mhmohona! 👋
> 
> I noticed the tests appear to be failing when running in the CI. Did notice also though that the screenshot in the PR description shows the tests do run OK when `pytest` is ran within `src/scribe_data/cli`. Does running `pytest` in the root of the project still run OK too?

Thank you @wkyoshida for looking into this. The thing is this tests passes when everything is in one place. I mean all the files, total.py, list.py, query.py is in one branch, then these tests works without any error. Both unittest and Pytest passes.

**Comment by andrewtavis** [comment]
Ok so the tests are passing, but then a question I have is when are the tests that we have written here ran? Is it possible to move the tests for the CLI into the [Scribe-Data/tests](https://github.com/scribe-org/Scribe-Data/tree/main/tests) directory such that they're ran as a part of the pytest suite?

**Comment by andrewtavis** [comment]
Would you be able to make this change, @mhmohona? Let us know if you have any questions!

**Comment by mhmohona** [comment]
> Ok so the tests are passing, but then a question I have is when are the tests that we have written here ran? Is it possible to move the tests for the CLI into the [Scribe-Data/tests](https://github.com/scribe-org/Scribe-Data/tree/main/tests) directory such that they're ran as a part of the pytest suite?

I will work on it.

**Comment by andrewtavis** [comment]
@mhmohona, just updated both your branches as there have been tons of changes recently. Let us know if there's anything we can do to support with these!

**Comment by mhmohona** [comment]
@andrewtavis this PR is up for review now.

File: tests/cli/test_list.py [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+        ]
+        mock_print.assert_has_calls(expected_calls)
+
+    def test_list_data_types_invalid_language(self):
+        with self.assertRaises(ValueError):
+            list_data_types("InvalidLanguage")
+
+    def test_list_data_types_no_data_types(self):
+        with self.assertRaises(ValueError):
+            list_data_types("Klingon")
```
Comment (andrewtavis) [inline tests/cli/test_list.py:94]: Maybe one day could be a language 😅 Wikidata does have data for it 😇 But fine for now :)

**Review by andrewtavis — APPROVED** [review]
Thanks for writing all of these tests, @mhmohona! Great that the Scribe community is focussing so much more on tests these days 😊

File: tests/cli/test_list.py [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+        mock_print.assert_has_calls(expected_calls)
+
+    @patch("builtins.print")
+    def test_list_data_types_all_languages(self, mock_print):
+        list_data_types()
+        expected_calls = [
+            call(),
+            call('Available data types: All languages'),
+            call('-----------------------------------'),
+            call('emoji-keywords'),
```
Comment (wkyoshida) [inline tests/cli/test_list.py:62]: Hmm - just noting something down here that came to mind just now..

Concerning the discussion we had on providing package options without PyICU and consequently the emoji functionality - omitting `emoji-keywords` as an available data type in the command output will probably be a good idea. With that, testing that the appropriate options are getting outputted given whether with/without PyICU might be good too.

...but! That's for when we get there :wink:

**Pull request merged**